Dockerfile

```diff
@@ -1,9 +1,9 @@
 # builder image
-FROM golang
+FROM golang:1.12.9
```

Comment:
Setting this to tag `1.12` will ensure we always take the latest without having to continually update the patch number